src/extension/index.ts

```diff
@@ -100,6 +104,10 @@ export function activate(context: vscode.ExtensionContext) {
             new ChatPanelProvider(context)
         )
     );
+
+    // TODO: No need to refresh the token every time.
+    // If the token is still valid, there is no need to refresh it.
+    refreshToken();
```

Comment:
We need to pay attention to the case where tokens expire while the user is using the app.

Reply:
I think we can do it in the next PR as an improvement.